src/AutoMapperSamples.EF/AutoMapperSamples.EF.csproj

```diff
@@ -0,0 +1,141 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Hi!
Basically this is only a sample project showing how to use EF with AutoMapper.
The only thing I really added in order to support EF6 ist the
AutoMapperSamples.OData.EntityFrameworkCompatibilityVisitor 
which maps "Queryable.LongCount" methodcallexpressions to "Queryable.Count" expressions as the former are not supported by EF. Should I add the Visitor to AutoMapper.EF?